app/models/screening_list/mappable.rb

```diff
@@ -36,7 +36,11 @@ def self.included(klass)
                                          name: { type: 'string', analyzer: 'standard_asciifolding_nostop' },
                                          sort: { type: 'string', analyzer: 'keyword_lowercase' } } },
 
-            alt_names:               { type: 'string', analyzer: 'standard_asciifolding_nostop' },
+            alt_names:               { type:   'multi_field',
+                                       fields: {
+                                         alt_names: { type: 'string', analyzer: 'standard_asciifolding_nostop' },
+                                         sort:      { type: 'string', analyzer: 'keyword_lowercase' } } },
```

Comment:
I went with the same naming as the 'name' field and used 'sort'. Is that the best name for this?